mne/viz/evoked.py

```diff
@@ -828,6 +894,38 @@ def plot_evoked_image(evoked, picks=None, exclude='bads', unit=True, show=True,
         resets the scale. Up and down arrows can be used to change the
         colormap. If 'interactive', translates to ``('RdBu_r', True)``.
         Defaults to ``'RdBu_r'``.
+    mask_cmap : matplotlib colormap | (colormap, bool) | 'interactive'
+        The colormap chosen for masked parts of the image (see below), if
+        `mask` is not ``None``. If None, `cmap` is reused. Defaults to
+        ``Greys``. Not interactive. Otherwise, as `cmap`.
+    mask : ndarray | None
+        An array of booleans of the same shape as the data. Entries of the
+        data that correspond to ```False`` in the mask are plotted
+        transparently. Useful for, e.g., masking for statistical significance.
+
+        .. versionadded:: 0.16
+    alpha : float
+        A float between 0 and 1. If `mask` is not None, this sets the
+        alpha level (degree of transparency) for the masked-out segments.
+        I.e., if 0, masked-out segments are not visible at all.
+        Defaults to .5.
+
+        .. versionadded:: 0.16
+    do_mask: None | bool
+         If ``True``, mask the image. If ``False``, do not apply mask.
+         If ``None`` and `mask` is not ``None``, defaults to drawing a mask.
+
+         .. versionadded:: 0.16
+    do_contour : None | bool
+         If ``True``, draw a contour surrounding the masked area.
+         If ``False``, do not add contour. If ``None`` and `mask` is not
+         ``None``, defaults to drawing a contour.
+
+         .. versionadded:: 0.16
+    colorbar : bool
+        If True, plot a colorbar. Defaults to True.
```

Comment:
how about we keep only 3 params: mask, mask_cmap, colorbar?
so keep the contour all the time and I don't see the need of do_mask...

Reply:
I know it's a lot of parameters, but I think Jean-Remi usually plots only contours, and the EEGLAB/FieldTrip style is to plot only the mask, and you like both ... I think people would expect it.

And I wouldn't think this will only be used for masking significance.

How about a vote? :D

Reply:
These are indeed a lot of arguments, but I think we should support these options with sensible defaults. It's a bit ugly though because what if `mask is not None` but both `do_mask` and `do_contour` are both `False`? What about combining these options to one argument `mask_style`, which could be `'mask'`, `'contour'` or `'both'`?